tests/infra_test.go

```diff
@@ -975,3 +971,34 @@ func getKeysFromMetrics(metrics map[string]float64) []string {
 	sort.Strings(keys)
 	return keys
 }
+
+func checkError(err error) error {
+	var e *neturl.Error
+	if errors.As(err, &e) && e.Timeout() {
+		return nil
+	} else {
+		return err
+	}
+}
+
+func waitHttpResponse(errorsChan chan error, concurrency int) (int, error) {
+	expectedErrorsCount := 0
+	var err error
+	for ix := 0; ix < concurrency; ix++ {
+		select {
+		case err = <-errorsChan:
+			if err != nil {
+				err = checkError(err)
+				if err != nil {
+					break
+				}
+				expectedErrorsCount += 1
+			}
```

Comment:
i count it only if its an expected error, in case its unexpected error its a fatal and we can break out
i dont need to create a list of error pools, just to return the first offender
we dont need to count timeout ones, i will also remove it because the comment of 
https://github.com/kubevirt/kubevirt/pull/4306#discussion_r503234748
even that i think its good enough imo.

Reply:
updated the func as we spoke